domains/cas.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "lil bio",
+    "repo": "https://github.com/caswantsanap/bio",
+    "owner": {
+        "username": "caswantsanap",
+        "email": "caswantsanap@gmail.com"
+    },
+    "record": {
+        "CNAME": "caswantsanap.github.io\bio"
```

CNAME can't contain `/`
```suggestion
        "CNAME": "caswantsanap.github.io"
```